tests/test_docker_cat.py

```diff
@@ -0,0 +1,539 @@
+        """As a SonarQube user, I want the plugins listed in the README
+        to be installed on the server so that I can use them.
+        """
+        required_plugins = (
+            ("C++ (Community)", "1.3.1 (build 1807)"),
+            ("Checkstyle", "4.21"),
+            ("Cobertura", "1.9.1"),
+            ("Findbugs", "3.11.0"),
+            ("Git", "1.8 (build 1574)"),
+            ("GitHub Authentication for SonarQube", "1.5 (build 870)"),
+            ("JaCoCo", "1.0.2 (build 475)"),
+            ("LDAP", "2.2 (build 608)"),
+            ("PMD", "3.2.1"),
+            ("Rules Compliance Index (RCI)", "1.0.1"),
+            ("SAML 2.0 Authentication for SonarQube", "1.2.0 (build 682)"),
+            ("Sonar Frama-C plugin", "2.1.1"),
+            ("Sonar i-Code CNES plugin", "2.0.2"),
+            ("SonarC#", "7.15 (build 8572)"),
+            ("SonarCSS", "1.1.1 (build 1010)"),
+            ("SonarFlex", "2.5.1 (build 1831)"),
+            ("SonarGo", "1.1.1 (build 2000)"),
+            ("SonarHTML", "3.1 (build 1615)"),
+            ("SonarJS", "5.2.1 (build 7778)"),
+            ("SonarJava", "5.13.1 (build 18282)"),
+            ("SonarKotlin", "1.5.0 (build 315)"),
+            ("SonarPHP", "3.2.0.4868"),
+            ("SonarPython", "1.14.1 (build 3143)"),
+            ("SonarQube CNES CXX Plugin", "1.1"),
+            ("SonarQube CNES Export Plugin", "1.2"),
+            ("SonarQube CNES Python Plugin", "1.3"),
+            ("SonarQube CNES Report", "3.2.2"),
+            ("SonarQube CNES Scan Plugin", "1.5"),
+            ("SonarRuby", "1.5.0 (build 315)"),
+            ("SonarScala", "1.5.0 (build 315)"),
+            ("SonarTS", "1.9 (build 3766)"),
+            ("SonarVB", "7.15 (build 8572)"),
+            ("SonarXML", "2.0.1 (build 2020)"),
+            ("Svn", "1.9.0.1295")
+        )
+        sonar_plugins = requests.get(f"{self.CAT_URL}/api/plugins/installed").json()['plugins']
+        installed_plugins = { plugin['name']: plugin['version'] for plugin in sonar_plugins }
+        for name, version in required_plugins:
+            # Hint: if this test fails, one or more plugins may be missing or installed with an outdated version
+            assert installed_plugins[name] == version
+
+    def test_check_qg(self):
+        """As a SonarQube user, I want the SonarQube server to have the CNES
+        Quality Gate configured and set as default so that I can use it.
+        """
+        quality_gates = requests.get(f"{self.CAT_URL}/api/qualitygates/list").json()['qualitygates']
+        cnes_quality_gates = [ gate for gate in quality_gates if gate['name'] == "CNES" ]
+        # Hint: if one of these tests fails, the CNES Quality Gate may not have been added correctly, check the container logs
+        assert cnes_quality_gates # not empty
+        assert cnes_quality_gates[0]['isDefault']
+
+    def test_check_qp(self):
+        """As a SonarQube user, I want the SonarQube server to have the
```

Comment:
Codacy found an issue: [1 blank line required between summary line and description (found 0) (D205)](https://app.codacy.com/gh/cnescatlab/docker-cat/pullRequest?prid=6253839)